jcache/embedded/src/test/java/org/infinispan/jcache/JCacheManagerUriNotHierarchicalTest.java

```diff
@@ -0,0 +1,65 @@
+package org.infinispan.jcache;
```

Comment:
Could you please move those 2 tests into `JCacheConfigurationTest` ? After all, specifying configuration file is a configuration thing :)